pkg/kor/exceptions/replicasets/replicasets.json

```diff
@@ -0,0 +1,9 @@
+{
+  "exceptionReplicaSets": [
```

Comment:
Should we be keeping empty replicasets?

Reply:
When I come to think about it, except for optional rollback (less common with default installation ones), there's no much added value in preserving unused ReplicaSets. I'll revert this addition, encouraging users to remove them and free up etcd space.